src/screens/ChatScreen/SignedChatScreen.tsx

```diff
@@ -97,7 +97,7 @@ const SignedChatScreen = () => {
         <ChatDetailHeader
           channel={selectedChannel}
           onAvatarPress={exitedGroup ? null : () => goToChatInfoPage()}
-          onBackPress={goBackFromChatScreen}
+          onBackPress={() => goBackFromChatScreen(SIGNED)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the `onBackPress` prop of `ChatDetailHeader` from `goBackFromChatScreen` to `() => goBackFromChatScreen(SIGNED)` is significant. This change implies that the `SIGNED` parameter is now being passed to the `goBackFromChatScreen` function. Ensure that this function can handle this new parameter correctly and that it doesn't introduce any unexpected behavior or side effects. If the function `goBackFromChatScreen` does not expect a parameter, this could lead to potential issues.
```diff
- onBackPress={goBackFromChatScreen}
+ onBackPress={() => goBackFromChatScreen(SIGNED)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->